packages/plugins/RedPacket/src/SiteAdaptor/RedPacketActionButton.tsx

```diff
@@ -146,7 +147,7 @@ export const RedPacketActionButton = memo(function RedPacketActionButton(props:
                 redpacketStatus === FireflyRedPacketAPI.RedPacketStatus.Refund
             }
             size="large">
-            {statusToTransMap[redpacketStatus]}
+            <span>{statusToTransMap[redpacketStatus]}</span>
```

Comment:
What's the difference?

Reply:
Ensure content and background have different opacity